### PR TITLE
Ex: Making a Toast. #5

### LINKED ISSUE #5: Ex: Making a Toast
- [x] Description of the exercise
- [x] Tasks
- [x] Tasks

### PR DESCRIPTION
**Pull request by varya**

### CODE REVIEW

File: slides/ds101-worskhop.mdx [hunk 1]
```diff
@@ -15,9 +15,13 @@ export const theme = { ...CDTheme };
 
 ---
 
-# Backing a toast
-
-## Ice breaker
+<Center>
```
Comment (okonet) [inline slides/ds101-worskhop.mdx]: It's using `<Center>` by default afaik so you can remove it

**Review by okonet — CHANGES_REQUESTED** [review]

File: slides/ds101-worskhop.mdx [hunk 2]
```diff
@@ -43,6 +47,36 @@ We had a lot of fun!
 
 ---
 
+# Making a Toast 1/3
+
+Individual task: 5 stages of making a toast  in the morning.
```
Comment (okonet) [inline slides/ds101-worskhop.mdx]:
```suggestion
Individual task: 5 stages of making a toast in the morning.
```

File: slides/ds101-worskhop.mdx (hunk 1 above)
Comment (varya) [reply slides/ds101-worskhop.mdx]: Removed

File: slides/ds101-worskhop.mdx (hunk 2 above)
Comment (varya) [reply slides/ds101-worskhop.mdx]: Fixed

File: slides/ds101-worskhop.mdx [hunk 3]
```diff
@@ -27,6 +27,26 @@ export const theme = { ...CDTheme };
 
 ---
 
+# Schedule
```
Comment (varya) [inline slides/ds101-worskhop.mdx]: This came here after rebase :-( Somehow it is in diff even though the master already has this content.

Comment (okonet) [reply slides/ds101-worskhop.mdx]: git log says:

```
* 2f12b49 - (HEAD -> making-a-toast, origin/making-a-toast) Update slides/ds101-worskhop.mdx (19 hours ago) <Andrey Okonetchnikov>
* 7664983 - Schedule (19 hours ago) <Varya Stepanova>
* b2a4495 - Review changes (19 hours ago) <Varya Stepanova>
* 4a6994c - Ex: Making a Toast. #5 (19 hours ago) <Varya Stepanova>
*   74a3882 - (origin/master, origin/HEAD, master) Merge pull request #19 from component-driven/schedule (19 hours ago) <Varya Stepanova>
|\  

```

**Comment by okonet** [comment]
I had to force push so please don't pull but reset instead

File: slides/ds101-worskhop.mdx [hunk 4]
```diff
@@ -27,9 +27,11 @@ export const theme = { ...CDTheme };
...
 ---
 
-# Backing a toast
-
-## Ice breaker
+<Figure
+src={require("./assets/toast.jpg")}
+alt="Yammy"
+caption="Making a Toast"
+></Figure>
```
Comment (okonet) [inline slides/ds101-worskhop.mdx]:
```suggestion
<Figure
src={require("./assets/toast.jpg")}
alt="Yammy"
caption="Making a Toast"
/>
```

**Review by okonet — APPROVED** [review]

**Pull request merged**